config-connector/tests/README.md

```diff
@@ -0,0 +1,158 @@
+# Config Connector Solutions Testing CLI
+
+## Introduction
+
+This folder contains the Go CLI and testcases for testing the Config Connector
+Solutions defined in [../solutions](../solutions) folder.
+
+*  **[ccs-test/](./ccs-test/)** - Go CLI
+*  **[testcases/](./testcases/)** - Testcases for each solution. If has
+   the same folder structure as the solutions, i.e. if the solution is under 
+   <code>../solutions/<b>iam/kpt/member-iam/</b></code>, then the corresponding
+   testcases should be under <code>./testcases/<b>iam/kpt/member-iam/</b></code>
+
+## Requirements
+
+*  [Go](https://golang.org/doc/install)
+*  [kpt](../solutions/README.md#kpt)
+*  [helm](../solutions/README.md#helm)
+*  a working Kubernetes cluster with Config Connector [installed and 
+   configured](https://cloud.google.com/config-connector/docs/how-to/install-upgrade-uninstall)
+
+## Consumption
+
+1.  Clone GoogleCloudPlatform/cloud-foundation-toolkit repository under your `$GOPATH`:
+  
+    ```
+    git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+    ```
+
+1.  Go to the tests folder:
+
+    ```
+    cd cloud-foundation-toolkit/config-connector/tests
+    ```
+
+1.  Install ccs-test CLI:
+
+    ```
+    go install ./ccs-test
+    ```
+
+1.  Set the environment variables for the tests.
+
+    1.  Make a copy of the environments template file
+        ([./testcases/environments.template](./testcases/environments.template)) :
+
+        ```
+        cp ./testcases/environments.template ./testcases/environments.yaml
+        ```
+
+    1.  Edit the environments file
+        ([./testcases/environments.yaml](./testcases/environments.yaml)) to
+        update the environment variables. Use **any command or editing tool**
+        you prefer. E.g. you can use `sed` command:
+
+        ```
+        # YOUR_PROJECT_ID should be the project ID that your default namespace
+        # is annotated with.
+        sed -i 's/${DEFAULT_PROJECT_ID?}/[YOUR_PROJECT_ID]/g' ./testcases/environments.yaml
+        ```
+
+        **Note:** Please remember to set **ALL** the environment variables.
+
+1.  Follow the README of each solution to configure permissions for the
+    cnrm-system service account, and enable necessary APIs.
+
+    **Note:** This step will be automated in the upcoming changes.
+
+## How to run the tests?
+
+**Note:** Currently we only support testing one solution each time by setting
+the relative path of the solution using `--path` or `-p` flag.
+
+**Note:** Currently we only support testing kpt solutions.
+
+Under the [tests](.) folder, run a test by providing the relative path:
```

Comment:
Any comment on what to expect/not to expect from the run, how long it should run and how to tell if it passed or failed?

Reply:
Done.